spring-pulsar-docs/src/main/asciidoc/reactive-quick-tour.adoc

```diff
@@ -45,7 +45,7 @@ public class ReactiveSpringPulsarHelloWorld {
 
     @Bean
     ApplicationRunner runner(ReactivePulsarTemplate<String> pulsarTemplate) {
-        return (args) -> pulsarTemplate.send("hello-pulsar-topic", "Hello Reactive Pulsar World!").subscribe();
+        return (args) -> pulsarTemplate.send("hello-pulsar-topic", Flux.just(MessageSpec.of("Hello Reactive Pulsar World!"))).subscribe();
```

Comment:
I think the previous sample use w/ single message may be less to digest on the quick-tour.